### ISSUE TITLE
StaxRip 2.1.3.0 QSVEnc error i cant use profile main444 QSVEncC.exe finished with error!

### ISSUE THREAD

**Issue by YinnVI** [staxrip/staxrip#234]
```
Error Video encoding (2.1.3.0)

Video encoding returned error exit code: 1 (0x1)

It's unclear what the exit code means, in case it's a Windows system error then it possibly means:

Incorrect function


--------------------------- Video encoding ---------------------------

QSVEnc 4.03

C:\Srip\2.1.3.0\Apps\Encoders\QSVEnc\QSVEncC64.exe --codec hevc --profile main444 --la-quality slow --bframes 5 --ref 5 --direct-bias-adjust --mbbrc --output-buf 128 --input-buf 3 --mfx-thread 3 --output-thread 1 --colormatrix bt2020c --colorprim bt2020 --colorrange full --transfer bt2020-12 --fullrange --d3d11 --sao all --ctu 64 --no-deblock --tskip --icq 25 -i "D:\Torrent\[ReinForce] Itai (BDRip 1920x1080 x264 FLAC)\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_temp\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_new.vpy" -o "D:\Torrent\[ReinForce] Itai (BDRip 1920x1080 x264 FLAC)\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_temp\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_new_out.hevc"

Error: Invalid value "main444" for "--profile"
  Option value should be one of below...
    For H.264/AVC
      auto, Baseline, Main, High
    For H.265/HEVC
      auto, main, main10
QSVEncC.exe finished with error!

StaxRip.ErrorAbortException: Video encoding returned error exit code: 1 (0x1)

It's unclear what the exit code means, in case it's a Windows system error then it possibly means:

Incorrect function


--------------------------- Video encoding ---------------------------

QSVEnc 4.03

C:\Srip\2.1.3.0\Apps\Encoders\QSVEnc\QSVEncC64.exe --codec hevc --profile main444 --la-quality slow --bframes 5 --ref 5 --direct-bias-adjust --mbbrc --output-buf 128 --input-buf 3 --mfx-thread 3 --output-thread 1 --colormatrix bt2020c --colorprim bt2020 --colorrange full --transfer bt2020-12 --fullrange --d3d11 --sao all --ctu 64 --no-deblock --tskip --icq 25 -i "D:\Torrent\[ReinForce] Itai (BDRip 1920x1080 x264 FLAC)\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_temp\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_new.vpy" -o "D:\Torrent\[ReinForce] Itai (BDRip 1920x1080 x264 FLAC)\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_temp\[ReinForce] Itai 10 (BDRip 1920x1080 x264 FLAC)_new_out.hevc"

Error: Invalid value "main444" for "--profile"
  Option value should be one of below...
    For H.264/AVC
      auto, Baseline, Main, High
    For H.265/HEVC
      auto, main, main10
QSVEncC.exe finished with error!


   at StaxRip.Proc.Start() in D:\Projekte\VB\staxrip\General\Proc.vb:line 372
   at StaxRip.QSVEnc.Encode() in D:\Projekte\VB\staxrip\Encoding\QSVEnc.vb:line 95
   at System.Threading.Tasks.Parallel.<>c__DisplayClass4_0.<Invoke>b__0()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at StaxRip.GlobalClass.ProcessJob(String jobPath) in D:\Projekte\VB\staxrip\General\GlobalClass.vb:line 302
```

**Comment by 44vince44** [comment]
Thanks for pointing that out @YinnVI 
This is the list of supported profiles in QSVEnc 4.03
                                - H.264: auto(default), Baseline, Main, High
                                - HEVC:  auto(default), main, main10
                                - MPEG2: auto(default), Simple, Main, High

So you shouldn't use Main444 it's not a profile of QSVEnc.

@stax76 that would mean the Main444 profile must be removed from list of selectable profiles for QSVEnc Intel h264 AND Intel h265. Am I Right ?

**Comment by stax76** [comment]
There is probably something wrong somewhere because it's still in the online docs:

https://github.com/rigaya/QSVEnc/blob/master/QSVEncC_Options.en.md#--profile-string

**Comment by Dendraspis** [comment]
> There is probably something wrong somewhere because it's still in the online docs:
> https://github.com/rigaya/QSVEnc/blob/master/QSVEncC_Options.en.md#--profile-string

Indeed, but the source code seems to have no implementation of *main444 - Profile*.

**Comment by stax76** [comment]
I've created a ticket for it.

https://github.com/rigaya/QSVEnc/issues/25

**Comment by 44vince44** [comment]
Rigaya confirmed that Main444 is **not supported** and added a commit to remove it from online docs.
So I guess it has to be removed from straxrip as well !

**Comment by stax76** [comment]
I've fixed and committed it, so it will be available in the next build 2.1.3.3.
